impl/src/context.rs

```diff
@@ -92,7 +95,7 @@ pub struct CrateDependencyContext {
   // build_data_dependencies can only be set when using cargo-raze as a library at the moment.
   pub build_data_dependencies: Vec<BuildableDependency>,
   pub dev_dependencies: Vec<BuildableDependency>,
-  pub aliased_dependencies: Vec<DependencyAlias>,
+  pub aliased_dependencies: BTreeMap<String, DependencyAlias>,
```

Comment:
Can you comment on the meaning of the string here?

Not  necessary, but a struct, holding a string may provide more meaning long term if that's fairly straight-forward.

Reply:
I'm happy to work on this a bit more, but I might suggest that it would be easier to commit this change and iterate, or just let you change my patch to your satisfaction in a followup.

The pattern here is something I would do if I reviewed a coworker's code on a daily basis, and I wanted to make that process smoother going forward. But I think I'll only be sending cargo-raze patches here and there.

As for the CLAs, mine is signed. There was also a warning about this in #282 regarding @GregBowyer's sign-off.